astropy/cosmology/io/tests/test_ecsv.py

```diff
@@ -30,6 +31,14 @@ class ReadWriteECSVTestMixin(IOTestMixinBase):
     See ``TestCosmology`` for an example.
     """
 
+    @pytest.fixture
```

Comment:
Is this still needed with your most recent addition to just this while reading?

Reply:
I think it is. For a bare ``QTable.read()``, e.g. in ``test_tofrom_ecsv_instance``.

Reply:
Ah, makes sense. I think there is a lot to be said for having the units enabled automatically on `astropy.cosmology` import, but definitely for another PR.